src/Entity/ApplenewsArticle.php

```diff
@@ -163,10 +163,13 @@ public function getModifiedFormatted($type = 'medium', $format = NULL) {
    * @param string|null $format
    *   String format.
    *
-   * @return string
+   * @return string|null
    *   String formatted date
    */
   protected function formatDate($date, $type = 'medium', $format = NULL) {
```

Comment:
Just a thought: if we type hint the `$date` parameter then the calling code should check for this and therefore this method won't ever return NULL.

Reply:
it's just a string @juampynr

Reply:
Even with a string typehint, it's possible to pass in an empty string so I think we'd still want this check.